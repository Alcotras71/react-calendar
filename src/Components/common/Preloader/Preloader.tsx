import React from "react";
import preloader from "../../../assets/preloader/preloader.svg";

const Preloader = () => {
  return (
    <div>
      <img
        alt="preloader"
        src={preloader}
        style={{
          position: "absolute",
          top: "0%",
          left: "0%",
        }}
      />
    </div>
  );
};

export default Preloader;
