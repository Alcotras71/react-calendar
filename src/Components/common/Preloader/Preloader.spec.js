import React from "react";
import Preloader from "./Preloader";

describe("Preloader component", () => {
  it("should render Preloader component", () => {
    const component = shallow(<Preloader />);
    expect(component).toMatchSnapshot();
  });
});
