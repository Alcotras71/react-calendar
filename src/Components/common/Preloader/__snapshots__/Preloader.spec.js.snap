// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader component should render Preloader component 1`] = `
<div>
  <img
    alt="preloader"
    src="test-file-stub"
    style={
      Object {
        "left": "0%",
        "position": "absolute",
        "top": "0%",
      }
    }
  />
</div>
`;
